lib/slack/real_time/socket.rb

```diff
@@ -4,39 +4,65 @@ class Socket
       attr_accessor :url
       attr_accessor :options
 
+      attr_reader :driver
+
       def initialize(url, options = {})
         @url = url
         @options = options
+        @driver = nil
       end
 
       def send_data(data)
-        @ws.send(data) if @ws
+        driver.text(data) if driver
```

Comment:
Yep and that's actually something I use frequently, for the RTM client you need to be able to send back data all the time.